packages/payload/src/config/types.ts

```diff
@@ -1174,7 +1174,7 @@ export type EditConfigWithRoot = {
 }
 
 export type EditConfigWithoutRoot = {
-  [key: string]: EditViewConfig
+  [key: string]: EditViewConfig & Record<string, unknown>
```

Comment:
The unknown keys are not needed here, right?

```suggestion
  [key: string]: EditViewConfig
```

Reply:
Yep - fixed it